docs/CONTRIBUTING.md

```
@@ -493,107 +391,56 @@ STELLAR_REBALANCE_SECRET=<TESTNET_SIGNER_SECRET>
-- root `npm run format` when configured
+### How to contribute an ADR
 
-Missing optional scripts are reported as skips. Any configured command that exits non-zero blocks the commit or push with the failing command visible in terminal output.
+1. Copy `docs/adr/template.md` to a new file named `docs/adr/NNNN-my-decision-title.md`.
+2. Fill in the details (Context, Decision, Consequences).
+3. Submit as part of your Pull Request.
 
 ---
 
-## Further reading
+ ## Further reading
```

markdownlint-cli2 (0.22.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)